Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -184,6 +184,7 @@ public void ConfigureServices(IServiceCollection services)
         services.AddSingleton<IAzureServicePrincipalConfig, AzureServicePrincipalConfig>();
         services.AddSingleton<AzureManagementService>();
         services.AddSingleton<ProjectUsageService>();
+        services.AddSingleton<ProjectStorageConfigurationService>();
```

Comment:
We'll want this to be a scoped or transient service so that it stays within the same session/request